include/grpcpp/impl/codegen/client_callback.h

```diff
@@ -174,6 +181,23 @@ class ClientBidiReactor {
   }
   void StartWritesDone() { stream_->WritesDone(); }
 
+  /// Holds are needed if this stream has operations that take place on it after
+  /// StartCall but from outside one of the reactions (OnReadDone, etc). If a
+  /// stream still has a hold in place, its resources will not be destroyed even
+  /// if the status has already come in from the wire and there are currently no
+  /// active callbacks outstanding. Similarly, the stream will not call OnDone
+  /// if there are still holds on it. For example, if a StartRead or StartWrite
+  /// operation is going to be initiated from elsewhere in the application, the
+  /// application should call AddHold before StartCall.  If there is going to
+  /// be, for example, a read-flow and a write-flow taking place outside the
+  /// reactions, then call AddHold(2) before StartCall. When the application
+  /// knows that it won't issue any more Read operations (such as when a read
+  /// comes back as not ok), it should issue a RemoveHold(). It should also call
+  /// RemoveHold() again after it does StartWriteLast or StartWritesDone that
+  /// indicates that there will be no more Write operations.
+  void AddHold(int holds = 1) { stream_->AddHold(holds); }
```

Comment:
This interface allowing you to specify multiple holds without a symmetric RemoveHold(n) interface looks a little odd. So I looked up the version we're using as a template, and it has an AddHold interface as well as AddMultipleHolds, which I really prefer.

Reply:
So I've made it like our template version, but are you concerned about the lack of a RemoveMultipleHolds ? The original doesn't have that either, fwiw